plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupConfigurationManager.java

```diff
@@ -133,15 +133,12 @@ public void testDuplicates()
         dao.createResourceGroupsTable();
         dao.createSelectorsTable();
         dao.insertResourceGroup(1, "global", "1MB", 1000, 100, 100, null, null, null, null, null, null, ENVIRONMENT);
-        try {
-            dao.insertResourceGroup(1, "global", "1MB", 1000, 100, 100, null, null, null, null, null, null, ENVIRONMENT);
-            fail("Expected to fail");
-        }
-        catch (RuntimeException ex) {
-            assertTrue(ex instanceof UnableToExecuteStatementException);
-            assertTrue(ex.getCause() instanceof org.h2.jdbc.JdbcException);
-            assertTrue(ex.getCause().getMessage().startsWith("Unique index or primary key violation"));
-        }
+        H2ResourceGroupsDao finalDao = dao;
```

Comment:
The test should be split into two instead: testDuplicateRoots, testDuplicateGroups